imod/mf6/boundary_condition.py

```diff
@@ -77,6 +79,27 @@ def __init__(self, allargs: Mapping[str, GridDataArray | float | int | bool | st
         else:
             expand_transient_auxiliary_variables(self)
 
+    def set_repeat_stress(self, times: dict[np.datetime64,np.datetime64]) -> None:
+        """
+        Set repeat stresses: re-use data of earlier periods.
+
+        Parameters
+        ----------
+        times: Dict of datetime-like to datetime-like.
+            The data of the value datetime is used for the key datetime.
+        """
+        warnings.warn(
+            f"""{self.__class__.__name__}.set_repeat_stress(...) is deprecated.
+            In the future, add repeat stresses as constructor parameters. 
+            An object containing them can be created using 'get_repeat_stress', as follows:
+            repeat_stress = get_repeat_stress(repeat_periods) # args before provided to River.set_repeat_stress
```

Comment:
The main user unfriendly thing about ``get_repeat_stress`` is its location: stowed away in an obscure module. Add its location:

``from imod.mf6.utilities.package_utils import get_repeat_stress``

Furthermore:
We are still unsure about this utility location. So please also add the location of this utility might change in the future.

Reply:
done